### DISCUSSION TITLE
🔗 Hover Box-Shadow Button Widget (Duplicate of #30210)

### DISCUSSION THREAD

**Discussion by Andrenalina** [elementor/elementor#21171] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi team!

Button widget doesn't allow adding box-shadow property in hover state. It works perfectly in the normal state, but when you hover over it, it keeps the effect of the normal state.

### Describe the solution you'd like

It would be great if in both states a drop shadow could be assigned with different parameters. Thanks in advance.

Best regards! :)

**Comment by nicholaszein** [comment]
Hello there, @Andrenalina!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #30210

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

Kind regards
